src/components/Inventory/Inventory.js

```diff
@@ -57,6 +58,7 @@ const itemTypeCategoryMap = Object.freeze({
   [SCARECROW]: FIELD_TOOLS,
   [SPRINKLER]: FIELD_TOOLS,
   [STONE]: MINED_RESOURCES,
+  [TOOL_UPGRADE]: '',
```

Comment:
I'm taking this empty string to mean that `TOOL_UPGRADE` is treated as a class of items (in terms of the overall user experience) that has no `farmhand.item`s associated with it. I think it's reasonable to extend the item category system like this for now (but see below), but I'd like to see a clearer distinction between categories with items and categories without. I'd like to see some tweaks here:

  1. A comment explaining why `TOOL_UPGRADE` is different from the other categories.
  2. More of a nit, but consider changing the empty string to a `null` (it's functionally equivalent, but I do something similar in other parts of the codebase to more explicitly indicate something is defined, but not really useful or actionable).

-------------------------------

Not necessary for this PR, but in the future we may want to consider extending the item categories system to support other purchasable categories with separate maps/data structures. In other words, tacking on `TOOL_UPGRADE` to `itemTypeCategoryMap` works for now, but this pattern may not scale well once there's more than one exception case.

Reply:
fully agree with what you've said here! i did this as a sort of band-aid to fix some errors and wasn't sure what the best long term solution is. i'll make your #1 and 2 changes, but am happy to make more changes that better set us up for the future if you want

Reply:
i added a named `UPGRADES` category and things are working here. i think that cleans this up enough for now, but let me know what you think once you get back to this